docs/UserGuide.md

```diff
@@ -405,7 +430,7 @@ Expand the recipe and view the detailed steps in a bigger window.
 - An index number is required for the Command. Refer to the indexes displayed for each [Card](#33-cards) in the
   [Side Bar](#32-side-bar).
 
-### 4.3.8 Cook Recipe
+### 4.5.8 Cook Recipe
```

Comment:
It's a little hard understand the example if you did not show which recipe you are cooking, I suppose the image update is in another PR?

Reply:
Yes, all updates to screenshots and examples will be done in a separate PR

Reply:
Yep. Images will be in a separate PR. Someone feedbacked that there were too many examples, etc.